app/models/councillor.rb

```diff
@@ -6,6 +6,12 @@ class Councillor < ActiveRecord::Base
   validates :authority, :email, presence: true
   validates :image_url, format: { with: /\Ahttps/, message: "must be HTTPS" }, allow_blank: true
 
+  def initials
```

Comment:
This feels a bit unclear cause it's not just 'initials' it's also formatting them specifically for our view.

Should the formatting step to split into a view helper?